listenbrainz_spark/request_consumer/request_consumer.py

```diff
@@ -44,6 +44,9 @@ def get_result(self, request):
             logger.error('Bad query sent to spark request consumer: %s', json.dumps(request), exc_info=True)
             return None
 
+        logger.info('Query: %s' % query)
+        logger.info('Params: %s' % str(params))
```

Comment:
let's try and be consistent with our use of loggers. Have a read of the [logging documentation](https://docs.python.org/3/library/logging.html#logging.Logger.debug):

> The msg is the message format string, and the args are the arguments which are merged into msg using the string formatting operator.

This means that these lines should be `logger.info('Query: %s', query)`.

The main reason for this is so that the logger can skip doing string interpolation if the correct logging level isn't set.
I don't mind too much which way we do this - string interpolation should be pretty fast either way, however we _should_ be consistent in the way that we do it. I see other logger lines in this file which use the parameter format.

Reply:
Makes sense. Fixed now.